v3.3/getting-started/kubernetes/installation/rbac.yaml

```diff
@@ -58,6 +58,12 @@ rules:
       - namespaces
     verbs:
       - get
+  - apiGroups: [""]
+      resources:
+      - nodes/status
+      verbs:
+      - patch
```

Comment:
Do we need patch and update? I thought we just needed patch?

Reply:
yes only patch should be work because i use `PatchStatus`